sql/event_crypt.h

```diff
@@ -0,0 +1,12 @@
+#ifndef EVENT_ENCRYPTER_H
+#define EVENT_ENCRYPTER_H
+
+#include "basic_ostream.h"
```

Comment:
Please check this header list for pruning too: basic_ostream, rpl_constants, binlog_event?

Reply:
Will do, after the merge is completed I assume ?

Reply:
Yep